internal/service/instance/nginx_instance_service.go

```diff
@@ -271,3 +280,34 @@ func isFlag(vals []string) bool {
 func isKeyValueFlag(vals []string) bool {
 	return len(vals) == 2
 }
+
+func getLoadableModules(nginxInfo *Info) (modules []string, err error) {
```

Comment:
probably could just log the error instead of returning it since you aren't using the error for anything in the caller function.
Would be consistent then with the other functions in this file like https://github.com/nginx/agent/blob/0078c98ec0c18d29533e5dde1d9d89d3ffecd97b/internal/service/instance/nginx_instance_service.go#L243
and 
https://github.com/nginx/agent/blob/0078c98ec0c18d29533e5dde1d9d89d3ffecd97b/internal/service/instance/nginx_instance_service.go#L259